utils/bazel/llvm-project-overlay/lldb/BUILD.bazel

```diff
@@ -800,24 +801,20 @@ gentbl_cc_library(
     deps = ["//llvm:OptParserTdFiles"],
 )
 
+alias(
+    name = "gdb-server",
+    actual = select({
+        "@platforms//os:macos": ":debugserver",
+        "//conditions:default": ":lldb-server",
+    }),
+)
+
 cc_binary(
     name = "lldb",
     srcs = glob([
         "tools/driver/*.cpp",
         "tools/driver/*.h",
     ]),
-    data = [
```

Comment:
removing this just because it being here doesn't actually help lldb discover it at runtime, so we might as well just let users provide it themselves.